README.md

```diff
@@ -26,19 +26,14 @@ Mailhound is a simple server side script for receiving form posts and emailing t
 3. Install mailhound with `npm install -g mailhound`
 4. Add config variable exports to the end of your `~/.profile` file
   1. `export SMTP=smtp://postmaster:pass@smtp.server.com`
-  2. `export KEY:name@domain.com`
+  2. `export ADMIN=name@domain.com`
```

Comment:
Thanks for changing this! I noticed it last night and thought I'd have to get to it.